### ISSUE TITLE
 Expense - GBR appears briefly in LHN when Schedule submit is enabled and admin requests money

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#39923]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.61-0
**Reproducible in staging?:** Y
**Reproducible in production?:** N
**Found when validating PR :** https://github.com/Expensify/App/pull/38675
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Issue reported by:** Applause-Internal team

## Action Performed:
1. Go to staging.new.expensify.com
2. Create a new Collect workspace.
3. Go to Profile > Workspaces > Collect workspace.
4. Go to More features > Enable workflows.
5. Go to Workflows.
6. Enable Add approvals.
7. Also please ensure that "Delay submissions" is enabled.
8. Return to workspace chat.
9. Create a manual request.

## Expected Result:
Since "Schedule submit" is enabled in Step 7, there should be no GBR in LHN when admin requests money from the workspace chat.

## Actual Result:
When admin requests money from the workspace chat, GBR appears for a while, then disappears.
This issue only happens when admin requests money from the workspace chat, not when employee requests money from the workspace chat.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

https://github.com/Expensify/App/assets/115492554/b3193076-df24-4d7f-8cf4-e8ce3d2862bb

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by izarutskaya** [comment]
@CortneyOfstad I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors.

**Comment by izarutskaya** [comment]
We think this issue might be related to the #collect project.

**Comment by mountiny** [comment]
> For paid group policies, if harvesting is enabled, no GBR should appear neither on admin's nor on member's side

correct, the report will submit automatically so no GBR as nobody is expected to take action

> For free group policies, if I'm requesting money from my own workspace, I should see the GBR

For free, there is no concept of Draft reports, so they are always in Submitted state when created. In that case the admin would see GBR as they need to pay

**Comment by mountiny** [comment]
@arosiclair happy to take this over as I have reviewed the PR

**Comment by mountiny** [comment]
Actually as I was writing the request to cherry pick I think this does not have to be CPed its a minor bug that wont really influence users so we can leave this go with the normal process

**Comment by mountiny** [comment]
I have created an issue here to internally handle that clean up as I think we need to make this logic slightly more robust and it will be easier if internal person handles it https://github.com/Expensify/App/issues/39953

**Comment by mountiny** [comment]
I believe we can close this now
